fontforgeexe/wordlistparser.c

```diff
@@ -284,15 +284,21 @@ u_WordlistEscapedInputStringToRealString_readGlyphName(
 	    {
 		unichar_t* endptr = 0;
 		long unicodepoint = u_strtoul( glyphname+3, &endptr, 16 );
+                SplineChar* tmp = 0;
```

Comment:
Sort of. But `NULL` exists for a reason, and I think that some compilers complain about type.